stl/inc/random

```diff
@@ -490,6 +490,16 @@ _NODISCARD _Uint _Next_linear_congruential_value(_Uint _Prev) noexcept {
         // numeric_limits<result_type>::max() plus 1." That is: Just do the multiply
         // and let normal unsigned modulo take care of it
         return static_cast<_Uint>(static_cast<_Uint>(_Ax * _Prev) + _Cx);
+    } else if constexpr (_Cx == 0 && _Mx == 2147483647) {
+        // minstd_rand

```

Comment:
Let's also mention that this applies to `minstd_rand0` and that we're avoiding constant divisions.